loguru.hpp

```diff
@@ -2546,7 +2546,8 @@ namespace loguru
 				}
 			}
 			auto duration_sec = (now_ns() - _start_time_ns) / 1e9;
-			log(_verbosity, _file, _line, "} %.*f s: %s", SCOPE_TIME_PRECISION, duration_sec, _name);
+            auto buff = textprintf("%.*f s: %s", SCOPE_TIME_PRECISION, duration_sec, _name);
+			log_to_everywhere(1, _verbosity, _file, _line, "} ", buff.c_str());
```

Comment:
This adds an extra allocation, which is why I didn't do this in the first place. However, I don't think people need all the perf for log scopes, so LGTM. Hoever, please fix the indentation to match (tabs!) and I'll merge.

Oh, and thanks for your contribution!